lib/form-validation.js

```diff
@@ -6368,7 +6368,7 @@
   };
 
   validate.validators.numerical = function (value, options, attribute, attributes, globalOptions) {
-    const delocalisedValue = value != undefined ? value.replace(",", ".").replace(" ", "") : value;
+    const delocalisedValue = value != undefined ? value.replaceAll(",", ".").replaceAll(" ", "") : value;
```

Comment:
Voor de spaties kan ik het wel begrijpen, maar voor de komma's vind ik het wat vreemd. Is dit dan de decimal separator of komma van separator per drie cijfers? Beetje verwarrend in dat geval. Eventueel ook een test voor bijmaken die verklaart waarom die komma replacement nodig is.

Reply:
nee klopt, kan eigenlijk niet voorkomen

Reply:
bij deze aangepast